### ISSUE TITLE
Feature Request: keyboard navigation in left-hand tree pane

### ISSUE THREAD

**Issue by elisimpson** [openscopeproject/TrguiNG#145]
It would be great to be able to use the keyboard to navigate the tree pane the way you can in the old Transmission Remote GUI app. Using the arrow keys to move up/down and open/close nested trees.

**Comment by qu1ck** [comment]
I'm confused. By left hand tree pane you mean the filters? Old transmission gui app does not have a tree there, just a list of filters.

If you mean the files tab in details pane then keyboard navigation already works there.

**Comment by elisimpson** [comment]
Yes, the filter list on the left side of the UI (I wasn't sure what it is called). If you click on a status, directory, tracker or other item in the filter list, you cannot move up or down with the keyboard, you have to click again to move. As you said, keyboard nav works properly in the "files" tab at the bottom of the screen.

It's super cool that the directory filter area is a tree with collapsible nav, unlike the long list I have in the old transgui. It would be great to see keyboard nav working here too.

**Comment by qu1ck** [comment]
Note to self: to make this work the filter section ordering logic needs to be reworked. Right now it just uses flex order property but  keyboard navigation needs the filters to be actually sequential in the configured order, not just appear so using css.

**Comment by qu1ck** [comment]
Implemented in [v1.3.0](https://github.com/openscopeproject/TrguiNG/releases/tag/v1.3.0)
